p2p/src/protocol/channel.rs

```diff
@@ -189,8 +189,8 @@ impl syncmgr::SyncHeaders for Channel {
         debug!(target: self.target, "[sync] {}", &event);
 
         match &event {
-            syncmgr::Event::HeadersImported(ImportResult::TipChanged(tip, height, _)) => {
-                info!(target: self.target, "Block height = {}, tip = {}", height, tip);
+            syncmgr::Event::HeadersImported(ImportResult::TipChanged(block_header, height, _)) => {
+                info!(target: self.target, "BlockHeader = {:#?} Block height = {}",  block_header, height);
```

Comment:
I'd prefer if we didn't change what is logged here.

Reply:
Done